manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Restic",
+  "name": "Restic for all",
```

Comment:
Why change the name?

Reply:
This was a proposition for another configuration of Restic. If you wanted to keep the package as is, it means we would have 2 Restic packages on YunoHost. I thought that having two different names would help. But as it seems that you're agreeing to update the package for a widest usage, we can keep the original name.